pkg/apis/core/types.go

```diff
@@ -3736,27 +3736,27 @@ const (
 	IPv6Protocol IPFamily = "IPv6"
 )
 
-// IPFamilyPolicyType represents the dual-stack-ness requested or required by a Service
-type IPFamilyPolicyType string
```

Comment:
@thockin Your PR message says that the old name was left for compat but I can't see it being left behind anywhere. I don't think that's necessary, just making sure this PR isn't missing anything you intended to included.

Reply:
Look in the v1 file:

```diff
+// for backwards compat
+type IPFamilyPolicyType = IPFamilyPolicy
```

Reply:
Thanks!